packages/stores/src/providers/WebsiteStatusProvider.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { useSubscription } from '@deriv/api';
+import useStore from '../useStore';
+import { useWS } from '@deriv/shared';
+import merge from 'lodash.merge';
+
+const WebsiteStatusProvider = ({ children }: React.PropsWithChildren<unknown>) => {
+    const { data, is_subscribed, subscribe, unsubscribe } = useSubscription('website_status');
+    const {
+        client: { is_logged_in, is_logging_in, loginid },
+        website_status: { update },
+    } = useStore();
+    const WS = useWS();
+
+    React.useEffect(() => {
+        if (data) update(prev => merge(prev, data));
```

Comment:
@adrienne-deriv Not sure if we should use `merge` here or not 🤔 If we use it then when for example `p2p_config` is not in the response we will still keep the previous data for it, Should we? 🤔 Do we care if `p2p_config` is there or not anywhere? 🤔